libnavui-androidauto/src/main/java/com/mapbox/androidauto/placeslistonmap/PlacesListOnMapManager.kt

```diff
@@ -92,10 +89,18 @@ class PlacesListOnMapManager(
                 .distanceFormatterOptions
                 .unitType
         )
-    }
 
-    private fun onDetached() {
-        placesListItemMapper = null
+        coroutineScope.launch {
+            placeRecords.collect { placeRecords ->
+                _itemList.value = CarLocationProvider.getRegisteredInstance().lastLocation()
+                    ?.let { currentLocation -> placesListItemMapper?.mapToItemList(
```

Comment:
`placesListItemMapper` can be a local variable now and the property can be removed.

Reply:
oh yeah even better!